featuretools/primitives/standard/transform_primitive.py

```diff
@@ -1078,3 +1080,85 @@ def date_to_holiday(x):
             df = df.merge(holidays_df, how='left')
             return df.names.values
         return date_to_holiday
+
+
+class DistanceToHoliday(TransformPrimitive):
```

Comment:
Should we try to move some of this into a utility? Since it uses similar functionality to our other primitive (DateToHoliday)?

We did this with CountString & WhiteSpaceCount
- https://github.com/alteryx/nlp_primitives/blob/main/nlp_primitives/whitespace_count.py#L6
- https://github.com/alteryx/nlp_primitives/blob/main/nlp_primitives/count_string.py#L9

We have a primitive util file too:
- https://github.com/alteryx/featuretools/blob/main/featuretools/primitives/utils.py

Reply:
@gsheni to me, there seems to be enough difference between them that additional abstraction isn't really needed.

Reply:
Gotcha, sounds good.

Reply:
@gsheni upon second look I think you had a good point. Let me know what you think

Reply:
yep, looks good.